bodo/libs/_pinnable.h

```diff
@@ -107,6 +107,12 @@ struct pinnable_ptr {
     using difference_type = typename std::iterator_traits<T *>::difference_type;
     using iterator_category = std::random_access_iterator_tag;
 
+    /// @brief Needed for MSVC compatibility
+    static pinnable_ptr pointer_to(element_type &obj) {
+        throw std::runtime_error(
+            "pinnable_ptr: pointer_to not implemeneted yet.");
+    }
```

Comment:
I think it's potentially dangerous not to implement this since STL containers may call it. I have a suggestion below but may not work for pin/unpin properly.

https://en.cppreference.com/w/cpp/memory/pointer_traits/pointer_to
```suggestion
    /// @brief Needed for MSVC compatibility
    static pinnable_ptr pointer_to(element_type &obj) {
        pinnable_ptr ptr;
        ptr.base_ = &obj;
        ptr.offset_ = 0;
        return ptr;
    }
```

Reply:
My thought process was that if this breaks something with pinning/unpinning then that might be difficult to find, which is why I was hoping the exception would catch where it is being used improperly. We could only define this on Windows builds to ensure this doesn't break any existing code?